core/shared/src/test/scala/kantan/codecs/resource/ResourceIteratorTests.scala

```diff
@@ -28,7 +28,7 @@ class ResourceIteratorTests
     extends AnyFunSuite with ScalaCheckPropertyChecks with Matchers with VersionSpecificResourceIteratorTests {
   // - Tools -----------------------------------------------------------------------------------------------------------
   // -------------------------------------------------------------------------------------------------------------------
-  case class FailingIterator[A](iterator: Iterator[A], index: Int) {
+  case class FailingIterator[A](it: Iterator[A], index: Int) {
```

Comment:
This seems slightly unnecessary, why did you change this name?

Reply:
There was a compiler error, I don't remember what exactly but names were colliding.